freqtrade/constants.py

```diff
@@ -484,6 +486,41 @@
             },
             'required': ['process_throttle_secs', 'allowed_risk']
         },
+        'external_message_consumer': {
+            'type': 'object',
+            'properties': {
+                'enabled': {'type': 'boolean', 'default': False},
+                'producers': {
+                    'type': 'array',
+                    'items': {
+                        'type': 'object',
+                        'properties': {
+                            'name': {'type': 'string'},
+                            'url': {'type': 'string', 'default': ''},
+                            'ws_token': {'type': 'string', 'default': ''},
+                        },
+                        'required': ['name', 'url', 'ws_token']
+                    }
+                },
+                'wait_timeout': {'type': 'integer', 'minimum': 0},
+                'sleep_time': {'type': 'integer', 'minimum': 0},
+                'ping_timeout': {'type': 'integer', 'minimum': 0},
+                'remove_signals_analyzed_df': {'type': 'boolean', 'default': False},
```

Comment:
remove_signals_analyzed_df is not used throughout the codebase
so this either includes a typo - or is reduncant / no longer necessary.

Reply:
This should also be `remove_entry_exit_signals`